File: examples/python/automations/links/delete.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.outlook.delete(
    outlook_id="outlook_DDw72RgWgIfKFXGaWXcGu",
)

File: cookbook/node/automations/outlook/delete.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.outlook.delete({
  outlook_id: "outlook_DDw72RgWgIfKFXGaWXcGu",
});

console.log(JSON.stringify(automation, null, 2));